base/sparse/sparsematrix.jl

```diff
@@ -1405,7 +1405,6 @@ conj!(A::SparseMatrixCSC) = (broadcast!(conj, A.nzval, A.nzval); A)
 
 # TODO: The following definitions should be deprecated.
 ceil{To}(::Type{To}, A::SparseMatrixCSC) = ceil.(To, A)
-floor{To}(::Type{To}, A::SparseMatrixCSC) = floor.(To, A)
```

Comment:
Good question. I thought perhaps the docstring implies that e.g. `floor(Float32, 1.7) == 1.0f0` should yield `true` (though `floor(Float32, 1.7)` presently fails), and hence the absence of `<:Integer` constraints. Or is the docstring itself too loose? Thanks!